demisto_sdk/commands/content_graph/interface/neo4j/neo4j_graph.py

```diff
@@ -424,6 +425,18 @@ def find_uses_paths_with_invalid_toversion(
             self._add_relationships_to_objects(session, results)
             return [self._id_to_obj[result] for result in results]
 
+    def find_items_used_deprecated_items(self, file_paths: List[str]) -> List[dict]:
```

Comment:
please declare the corresponding abstract method in `demisto_sdk/commands/content_graph/interface/graph.py`